src/main/resources/META-INF/plugin.xml

```diff
@@ -36,7 +36,6 @@
         ]]>
     </change-notes>
 
-    <idea-version since-build="223.4884.69"/>
```

Comment:
is it on purpose?

Reply:
Yes. I moved the sinceBuild property to the build.gradle file instead. It's needed for the verification task, as a minimum version to verify against. When the plugin is being built, this property is injected here automatically (for that to happen, I removed the `updateSinceUntilBuild = false` line from the build.gradle file).